src/StartMenu.ts

```diff
@@ -296,64 +298,85 @@ export class AEntryPointList implements IEntryPointList {
   private updateList(data: INamedSet[]) {
     const that = this;
 
+    const headerText = this.desc.name === 'Genes'? 'Gene Sets' : `${this.desc.name.substr(0, this.desc.name.length - 1)} Panels`;
```

Comment:
you're right, I didn't think about that. So the alternative would be to add the headers to the plugin and then get the headers from the plugin in this method, right?